README.md

```diff
@@ -7,27 +7,7 @@ Carbon is a library of reusable [React](https://facebook.github.io/react/) compo
 See the [demo and documentation site](https://carbon.sage.com/) for live examples, or see [create-carbon-app](https://github.com/sage/create-carbon-app) to get started.
 
 ## Documentation
-
-### Getting Started
-
-* [Setting up your environment](docs/guides/setting-up-your-environment.md)
-* [Getting started](docs/guides/getting-started.md)
-
-### Guides
-
-* [Assets](docs/guides/assets.md)
-* [Immutable](docs/guides/immutable.md)
-* [Validations](docs/guides/validations.md)
-* [Decorators](docs/guides/decorators.md)
-* [Services](docs/guides/services.md)
-* [Integrating Carbon with other UI](docs/guides/integrating-with-other-ui.md)
-* [Testing changes to Carbon in your project](docs/guides/installing-unreleased-changes.md)
-* [Testing process and tools](cypress/README.md)
```

Comment:
Why we have deleted this guide?

Reply:
They were getting more and more out of date and didn't reflect the advice we've been giving in person. There is a plan to write new collaboration usage instructions but in the meanwhile it's better to remove these, rather than give wrong guidance.

Reply:
just FYI, we've updated `Testing process and tools` readme file